nncf/hw_configs/vpu.json

```diff
@@ -35,6 +35,9 @@
     "operations": [
         {
             "type": "Convolution",
+            "attributes": {
+                "adjust_padding": true
```

Comment:
The problem I see is that there is no connection of this attribute with the precision that it has an impact on.

Reply:
the precision would be not enough, need to connect with a specific quantization config:
```
 "bits": 4,
 "mode": "symmetric",
 "granularity": "pertensor"
 "signed": true
 ```
I supposed we won't need this feature for any other combinations in a future. It's a current workaround for a specific case of U4 activations, which most probably will be deleted. Do you really think we should invest more time to make it flexible for other combinations?